nano/node/node.cpp

```diff
@@ -3328,6 +3362,7 @@ nano::active_transactions::active_transactions (nano::node & node_a) :
 node (node_a),
 started (false),
 stopped (false),
+difficulty_cb (20),
```

Comment:
Can we initialize all the values with the publish_threshold with this constructor https://www.boost.org/doc/libs/1_61_0/doc/html/boost/circular_buffer.html#idp9066352-bb

Then we'll always have a minimum of the publish_threshold on all the samples.

Reply:
initialized with the threshold